public/sanda/practice/sanda4.html

```diff
@@ -6,22 +6,26 @@
 </head>
 <body>
 
-
 <script>
-    function myFunction1() {
-        console.log('Hello humans!')
+
+    function currentDate() {
+        console.log(new Date().toLocaleDateString());
     }
 
-    function myFunction2(x) {
-        console.log(x)
+    function getPowFive(numb) {
+        console.log(Math.pow(numb, 5));
     }
 
-    function myFunction3() {
-        myFunction1();
-        myFunction2(87);
+    function getSumAndDivide(a, b) {
+        console.log(a + b)/b;
```

Comment:
There is a problem on this line. 
Try calling `getSumAndDivide` function with parameters: 3, 4.
**Actual result**: 7
**Expected result:** _(three plus four) divided by four_ = **1.75**

The problem is in the way you use `console.log()` instruction. 
`log` is a method(function) of `console` object. This method may receive arguments and outputs their values in the console.
On this line you try to do something like this: `output 'a+b' in the console and then divide it's result by 'b'`
But you should make it do the following: `output result of '(a+b)/4' in the console`